samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -819,3 +912,18 @@ The sample uses the following providers to generate the advertising packet paylo
 * Google Fast Pair provider (:kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR`)
 
 The sample uses the Bluetooth device name provider (:kconfig:option:`CONFIG_BT_ADV_PROV_DEVICE_NAME`) provider to generate the scan response data.
+
+When the DFU functionality is enabled (:ref:`CONFIG_APP_DFU <CONFIG_APP_DFU>`), the sample uses the SMP GATT Service UUID provider.
+Depending on the Fast Pair advertising mode, the SMP UUID is placed in the:
+
+* Advertising data - when Fast Pair is in the discoverable mode
+* Scan response data - when Fast Pair is in the not discoverable mode
```

Comment:
Maybe we should align this with the line 169 ?

Reply:
sure, aligned, please verify